syntax/parse.go

```diff
@@ -47,6 +47,23 @@ func Parse(filename string, src interface{}, mode Mode) (f *File, err error) {
 	return f, nil
 }
 
+// ScanAndDiscard tokenizes the input data and discards the tokens.
+// Parameters are as for Parse.
+// It exists only for internal benchmarking purposes.
```

Comment:
It's unfortunate this needs to be part of the external API for benchmarking. Could the `"scan"` benchmark be moved to the `syntax` package instead?